faq/debugging.inc

```diff
@@ -238,8 +238,14 @@ assume that the following would immediately work:
 shell$ mpirun -np 4 xterm -e gdb my_mpi_application
 </geshi>
 
-Unfortunately, it likely _won't_ work.  Several factors must be
-considered:
+If running on a personal computer, this will probably work.
+You can also use " . aurchin_click("https://github.com/Azrael3000/tmpi") .
+"<code>tmpi</code></a> to launch the debuggers in separate [tmux]
+panes instead of separate [xterm] windows, which has the advantage of
+synchronizing keyboard input between all debugger instances.
+
+Unfortunately, the `tmpi` or `xterm` approaches likely _won't_ work
```

Comment:
Oh, sorry, I defaulted to Github markdown for my comment... I think it should be `[tmpi]` and `[xterm]` here in the FAQ markdown format, right?

Reply:
right, fixed.